interactive/commands.py

```diff
@@ -41,6 +42,23 @@ def create_repo(*, name, get_github_api=_get_github_api):
     return repo["html_url"]
 
 
+@transaction.atomic()
```

Comment:
There's a change in semantics in this refactor.  The was a reason we originally put the slack/email notifications inside a transaction - if we failed to notify slack, the db would rollback.

As it now stands, the db would commit, the slack/email notification could fail, and we would have no indication of a problem, and the user would be left hanging.

I think the transaction belongs as with statement around the create report and sending.